### PR TITLE
Set up a private community

### PR DESCRIPTION
**Pull request by jorge-campo**
Hi @cheny0 - I am sending this article for your review; "Set up a private community".

I also made some changes to "Set up channel permissions" (`set-up-channel-permissions.md`, CC/ @Fabiomorais87), but you can ignore these changes and focus only on the community content here https://github.com/status-im/help.status.im/pull/542/files#diff-5300703b4f8a8e91a10a7d164e6036febb0ddf878e9ab2651012aa6e6a7527c2R1-R48.

---

- 522 first draft -- ready for review
- 522 updates 'Set up channel permissions' title and references - Additional content fixes

### CODE REVIEW

File: docs/en/status-communities/set-up-a-private-community.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+!!! note
+    In addition to tokens, community owners can require [manual approval][manage-community-join-requests] to join their communities. Token-based permissions and manual approval work independently.
+
+Token requirements can be useful to ensure that your community members have a certain level of commitment or interest in the topic of the community. If you run an organization with an ENS domain, you can also set up a private community for holders of your ENS domain only (for example, alice.acme.eth or bob.acme.eth).
+
+## What to expect
+
+- Use the Become member token-based permission to set up a private community.
+- You can set up token-based permissions with existing tokens or tokens you have [minted][mint-tokens-for-your-community] and distributed via [airdrops][how-to-airdrop-tokens-in-status].
+- In a private community, members must always meet and keep the token requirements. Community members not holding the required tokens lost access to the community automatically.
```
Comment (cheny0) [inline docs/en/status-communities/set-up-a-private-community.md:27]: - In a private community, members must always meet and keep the token requirements. Community members not holding the required tokens lose access to the community automatically.

File: docs/en/status-communities/set-up-a-private-community.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Set up a private community
+
+Community owners customize permissions using three different variables: `Who holds`, `Is allowed to` and `In`.
+
+=== "Desktop"
+
+    1. From the navigation sidebar, click your community.
+    1. On top of the channel sidebar, click your community logo and then, click **Permissions**.
+    1. Click **Add permission**.
+    1. For the `Who holds` variable, select the tokens and amounts (for assets and collectibles) members must hold to join your community. You can choose and combine up to five Assets, Collectibles or ENS.
```
Comment (cheny0) [inline docs/en/status-communities/set-up-a-private-community.md:40]:
```suggestion
    1. For the `Who holds` variable, select the tokens and amounts (for assets and collectibles) members must hold to join your community. You can choose and combine up to five assets, collectibles or ENS names.
```
I suppose "assets, collectibles" are not proper nouns, and adding "names" to make the sentence more fluent.

**Review by cheny0 — COMMENTED** [review]
Hi @jorge-campo , I only have two comments :)

**Comment by jorge-campo** [comment]
Thanks for your review, @cheny0 -- I applied your corrections.

**Pull request merged**